ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -689,7 +689,7 @@ <h3 class="title">
                   >
                     <span>Drop Foreign Key</span>
                   </button>
-                  <button
+                  <!-- <button
```

Comment:
Dropping a fk is not waiting for the user to click on save and dropping it instantly